mission/simulator/opssat-gps-simulator/src/main/java/opssat/simulator/GPS.java

```diff
@@ -101,9 +102,9 @@ public OrbitParameters getPosition(Date time) {
     public static BigDecimal truncateDecimal(double x, int numberofDecimals) {
         // From: http://stackoverflow.com/questions/7747469/how-can-i-truncate-a-double-to-only-two-decimal-places-in-java
         if (x > 0) {
-            return new BigDecimal(String.valueOf(x)).setScale(numberofDecimals, BigDecimal.ROUND_FLOOR);
+            return new BigDecimal(String.valueOf(x)).setScale(numberofDecimals, RoundingMode.FLOOR);
```

Comment:
Are we sure this is strictly equivalent? And what do we gain in doing this?

Reply:
Yes. the method with BigDecimal it's deprecated.

` * @deprecated The method {@link #setScale(int, RoundingMode)} should
     * be used in preference to this legacy method.`